timed/employment/filters.py

```diff
@@ -13,6 +14,8 @@ class YearFilter(Filter):
     """Filter to filter a queryset by year."""
 
     def filter(self, qs, value):
+        if value in EMPTY_VALUES:
```

Comment:
the doc string should still come right after the method definition.

Although in this case actually as the doc string doesn't say anything really you can also remove it.